Composer/packages/types/src/creation.ts

```diff
@@ -6,3 +6,61 @@ export const nodeFeedKey = 'firstPartyNode';
 export const defaultFeeds = [nodeFeedKey, csharpFeedKey] as const;
 export type FeedName = typeof defaultFeeds[number];
 export type FeedType = 'npm' | 'nuget';
+
+export type TeamsManifest = {
```

Comment:
define then use, move the definition for types above where they are used.

Reply:
resolved